generator/templates/view.lkml

```diff
@@ -0,0 +1,69 @@
+view: {{ view }} {
+{% if (tables|length) == 1 %}
+  sql_table_name: `{{ tables[0]["table"] }}` ;;
+{% else %}
+  derived_table: {
+    sql:
+  {% for table in tables %}
+      SELECT
+        *{% if "channel" in (dimensions|map(attribute="name")) %} EXCEPT (channel){% endif %},
+        {{ table["channel"] | tojson }} AS channel,
+      FROM
+        `{{ table["table"] }}`
+    {% if not loop.last %}
+      UNION ALL
+    {% endif %}
+  {% endfor %}
```

Comment:
I think for this MVP we don't worry about channels and just include the `release` channel. Let's add that on as a next step.

The problem is unioning doesn't work for these channels. I can tackle adding those as a parameter.